### ISSUE TITLE
Choose the audio stream

### ISSUE THREAD

**Issue by alxmrs** [gotev/android-speech#22]
It would be nice to be able to choose the audio stream for the Speech instance. This would enable usage of the TTS system on bluetooth devices. 
Something like: 
```
Speech.getInstance().setAudioStream(AudioManager.STREAM_VOICE_CALL);
```
I'll submit a PR with this enhancement.

**Comment by alxmrs** [comment]
Added PR #23

**Comment by gotev** [comment]
Fixed by #23

**Comment by gotev** [comment]
Feature available in release `1.4.0`
